src/projectscene/view/timeline/timeformat.cpp

```diff
@@ -12,72 +12,86 @@ IntervalInfo TimeFormat::intervalInfo(TimelineContext* context)
 {
     IntervalInfo timeInterval;
     double zoom = context->zoom();
-    // NOTE: 24 is minimal px size between minorMinor ticks
+    // NOTE: 32 is minimal px size between minorMinor ticks
     // 0.000005 is distance [s] between minorMinor ticks
     // same goes for other time intervals
-    if (zoom > (24 / 0.000005)) { // 0.000005s ticks
+    if (zoom > (32 / 0.000005)) { // 0.000005s ticks
```

Comment:
@grliszas14 what do you think about converting this to lookup table?

example totally not written by llm
```
struct IntervalInfo {
    double minor;
    double major;
    double minorMinor;
    int digits;
};

const std::vector<std::pair<double, IntervalInfo>> zoomToIntervalInfo = {
    {32 / 0.000005, {0.00002, 0.0001, 0.000005, 6}},
    {28 / 0.00001, {0.00005, 0.0005, 0.00001, 6}},
    // ... other entries ...
};

IntervalInfo TimeFormat::intervalInfo(TimelineContext* context) {
    double zoom = context->zoom();

    // Find the appropriate interval info using binary search or linear search
    // based on the zoom level.
    auto it = std::lower_bound(zoomToIntervalInfo.begin(), zoomToIntervalInfo.end(), zoom,
                              [](const auto& pair) { return pair.first > zoom; });

    if (it == zoomToIntervalInfo.end()) {
        // Handle the case where zoom is larger than the largest zoom value in the table
        return zoomToIntervalInfo.back().second;
    } else {
        return it->second;
    }
}
```

Reply:
Sounds good for minutes&seconds, changed it. I'm not so sure if to apply it as well to beats&measures, I don't think we're gonna gain any clarity there